LICENSE

```diff
@@ -0,0 +1,202 @@
+
+                                 Apache License
```

Comment:
Why did you chose this license? If not for any special reason I suggest we use `MIT`, which is a straight forward and permissive license.